airgun/views/rhai.py

```diff
@@ -0,0 +1,138 @@
+from widgetastic_patternfly import Button, Dropdown
+from widgetastic.utils import ParametrizedLocator
+from widgetastic.widget import (
+    Checkbox,
+    GenericLocatorWidget,
+    ParametrizedView,
+    Table,
+    Text,
+    TextInput,
+    View
+)
+
+from airgun.views.common import BaseLoggedInView
+
+
+class Overview(BaseLoggedInView):
+    title = Text(".//h1[@class='page-title']//span")
+
+    @View.nested
+    class action_summary(View):
+        title = Text(".//h2[@class='title']")
```

Comment:
same here, why do you need `title` and `action_summary` view here? It contains zero of useful functionality